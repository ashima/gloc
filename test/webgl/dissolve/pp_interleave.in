void main() {
#if Z
gl_Position = vec4(Z);
#else
gl_Position = vec4(1.);
#endif
}
