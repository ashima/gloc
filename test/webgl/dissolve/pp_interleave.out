File 0, line 1, col 12 - 13:
ESSL parse error: '{'

Fatal: unrecoverable parse error (7)
